src/main/java/racingGame/business/CarFactory.java

```diff
@@ -0,0 +1,19 @@
+package racingGame.business;
```

Comment:
```suggestion
package racinggame.business;
```
패키지 이름은 전체가 소문자로 구현하는 것이 컨벤션